content/en/docs/Configuration/p8s-jaeger-grafana/prometheus.md

```diff
@@ -0,0 +1,183 @@
+
+```
+    - job_name: kubernetes-pods
+      metric_relabel_configs:
+      - action: drop
+        source_labels: [__name__]
+        regex: istio_agent_.*|istiod_.*|istio_build|citadel_.*|galley_.*|pilot_.*|envoy_cluster_[^u].*|envoy_cluster_update.*|envoy_listener_[^dh].*|envoy_server_[^mu].*|envoy_wasm_.*
+      - action: labeldrop
+        regex: chart|destination_app|destination_version|heritage|.*operator.*|istio.*|release|security_istio_io_.*|service_istio_io_.*|sidecar_istio_io_inject|source_app|source_version
+```
+
+Applying this configuration should reduce the number of stored metrics by about 20%, as well as reducing the number of attributes stored on many remaining metrics.
+
+
+
+### Metric Thinning with Crippling
+
+The section above drops metrics unused by Kiali. As such, making those configuration changes should not negatively impact Kiali behavior in any way. But some very heavy metrics remain. Some, or all of these metrics can also be dropped, but their removal will impact the behavior of Kiali.  This may be OK if you don't use the affected features of Kiali, or if you are willing to sacrifice the feature for the associated metric savings. In particular, these are "Histogram" metrics.  Istio is planning to make some improvements to help users better configure these metrics, but as of this writing they are still defined with fairly bad default "buckets", making the number of associated time-series quite large, and the overhead of maintaining and querying the metrics, intensive.  Each histogram actually is comprised of 3 stored metrics.  For example, a histogram named `xxx` would result in the following metrics stored into Prometheus:
+
+- `xxx_bucket`
+  - The most intensive metric, and is required to calculate percentile values.
+- `xxx_count`
+  - Required to calculate 'avg' values.
+- `xxx_sum`
+  - Required to calculate rates over time, and for 'avg' values.
+
+When considering whether to thin the Histogram metrics, one of the following three approaches is recommended:
+
+1. If the relevant Kiali reporting is needed, keep the histogram as-is.
+2. If the relevant Kiali reporting is not needed, or not worth the additional metric overhead, drop the entire histogram.
+3. If the metric chart percentiles are not required, drop only the xxx_bucket metric.  This removes the majority of the histogram overhead while keeping rate and average (non-percentile) values in Kiali.
+
+
+These are the relevant Histogram metrics:
+
+#### istio_request_bytes
+
+This metric is used to produce the `Request Size` chart on the metric tabs.  It also supports `Request Throughput` edge labels on the graph.
+
+- Appending `|istio_request_bytes_*` to the `drop` regex above would drop all associated metrics and would prevent any request size/throughput reporting in Kiali.
+- Appending `|istio_request_bytes_bucket` to the `drop` regex above, would prevent any request size percentile reporting in the Kiali metric charts.
```

Comment:
Today, it wouldn't prevent it but instead report empty right?

Reply:
That's true, I guess, although the semantics are pretty close.  Instead of changing the wording I'm adding the UI-side disabling/hiding, so I guess the wording will eventually be correct :-)

Reply:
I added all of your edits, good eyes.  I'm not sure why the htmlproofer is unhappy with things.